Windows/FFmpeg.autopkg

```diff
@@ -0,0 +1,82 @@
+configurations
+{
+   Toolset
+   {
+      key : "PlatformToolset";
+      choices: { v141 };
+   };
+}
+
+nuget
+{
+   nuspec
+   {
+      id = TechSmith.FFmpeg;
+      version: 0.0.6;
+      title: FFmpeg Library;
+      authors: { TechSmith Corporation };
```

Comment:
Should authors be FFmpeg?

Reply:
Yeah, good call. I will make that changes.